docs/ibc/upgrades/quick-guide.md

```diff
@@ -0,0 +1,44 @@
+# How to Upgrade IBC Chains and their Clients
+
+The information in this doc for upgrading chains is relevant to SDK chains. However, the guide for counterparty clients is relevant to any Tendermint client that enables upgrades.
+
+## Upgrading Chains
+
+
+### IBC Client Breaking Upgrades
```

Comment:
I think we should outline the types of upgrades that clients could encounter (planned, unplanned) and with breaking an non-breaking state machine changes. This doc should also document the recommended steps for each of the 4 combinations

Reply:
Yup good point. Currently unplanned upgrades are not supported. All of the cases given only apply if they are planned.

Will write this point in.